src/app/components/core/asset-readings/readings-graph/readings-graph.component.css

```diff
@@ -10,4 +10,17 @@
 
 [hidden] {
   display: none !important;
+}
+
+.summary {
+  width: auto;
+  min-height: 15vh;
+  max-height: 25vh;
+  overflow-y: scroll;
+  padding-top: 3px;
+  padding-bottom: 3px;
+}
+
+.modal-card {
+  width: 70%;
```

Comment:
Need to avoid the scrolling behavior within individual blocks of modal. Modal should have only one scroll bar.